portfolio/src/main/java/com/google/sps/servlets/AddCommentServlet.java

```diff
@@ -0,0 +1,50 @@
+package com.google.sps.servlets;
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
```

Comment:
this blank line is not needed. see https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing